src/homework10/Task2/Main.java

```diff
@@ -0,0 +1,36 @@
+package homework10.Task2;
+
+import java.util.Scanner;
+public class Main {
+    public static void main(String[] args) {
```

Comment:
The main idea was to did something like sorting, rearrangement. Not only execute print
 So, try to rewrite)

Reply:
Hello Andrii, I have resolved this task by using ArrayList, Collections and List with sorting. Please check https://github.com/Q3-23-Organization2/aqa-pet/pull/7/commits/171215e24b74c1ffa78bba26dc396ae4536b17d5 
Thank You.